---------------------
PatchSet 321 
Date: 2010/09/25 08:12:01
Author: millert
Branch: HEAD
Tag: (none) 
Log:
Use snprintf() not sprintf()

Members: 
	x2p/str.c:1.10->1.11 

Index: src/gnu/usr.bin/perl/x2p/str.c
diff -u src/gnu/usr.bin/perl/x2p/str.c:1.10 src/gnu/usr.bin/perl/x2p/str.c:1.11
--- x2p/str.c:1.10	Mon Oct 12 18:30:28 2009
+++ x2p/str.c	Sat Sep 25 15:12:01 2010
@@ -29,7 +29,7 @@
     GROWSTR(&(str->str_ptr), &(str->str_len), 24);
     s = str->str_ptr;
     if (str->str_nok) {
-	sprintf(s,"%.20g",str->str_nval);
+	snprintf(s,str->str_len,"%.20g",str->str_nval);
 	while (*s) s++;
     }
     *s = '\0';
